src/pages/index.js

```diff
@@ -66,53 +61,43 @@ const IndexPage = ({ data, pageContext, location }) => {
           >
             <div className={styles.introText}>
               <p>
-                New Relic Instant Observability (I/O) helps you monitor your
-                stack like a pro, without the burden of manual set up. Explore
-                an open source ecosystem of 500+ quickstarts—out-of-the-box
-                bundles of integrations, dashboards, and alerts—created by
-                observability experts around the world, vetted by New Relic, and
-                ready for you to install with one click.
-                <br />
-                <br />
-                Didn't find what you're looking for? You're always{' '}
-                <a href="https://github.com/newrelic/newrelic-quickstarts">
-                  welcome to add contributions
-                </a>{' '}
-                or improve existing quickstarts to help others instantly monitor
-                their services.
+                New Relic is an observability platform that helps you accurately
+                capture data critical to the success of your team. Whether
+                you're a developer installing your first agent or you're ready
+                to query ingested data, our docs site can help you find the best
+                solution for your performance needs:
```

Comment:
Appealing to different audience is great here, but I think there's a bit of whiplash going on.

The user lands on the dev site, but then instantly gets talked to about the docs site. That's the goal, but I think this intro should try to orient them a bit about why we're linking them over to the docs site AND specifically mention the docs site is a different site.

Reply:
maybe changing 'our' in the links to 'Quickstart' blahblahblah to 'the Quickstart blahblahblah on the New Relic docs site' instead. Or some kind of 'docs site' call out specifically.